nltk/test/unit/test_util.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for nltk.util.
+"""
+
+import unittest
+from nltk.util import everygrams
+
+
+class TestEverygrams(unittest.TestCase):
+
+    def setUp(self):
+        """Form test data for tests."""
+        self.test_data = 'a b c'.split()
```

Comment:
What if we set `test_data` to an iterable at this point, i.e. `iter('a b c'.split())`?

This would mean you can use the variable directly as is in your tests without having to call `iter` on it. You would also have to break up your first test method into 3 methods, which I actually think is a good thing.

Reply:
Done